packages/libs/components/src/plots/VolcanoPlot.tsx

```diff
@@ -150,8 +150,21 @@ function VolcanoPlot(props: VolcanoPlotProps) {
           so use caution when ordering the children (ex. draw axes before data).  */}
       <XYChart
         height={height ?? 300}
-        xScale={{ type: 'linear', domain: [xMin, xMax] }}
-        yScale={{ type: 'linear', domain: [yMin, yMax], zero: false }}
+        xScale={{
+          type: 'linear',
+          domain: independentAxisRange
```

Comment:
Will `xMin` and `xMax` ever be used if `independentAxisRange` is passed in? Similar question re: `yMin` and `yMax` wrt to `dependentAxisRange`.

I ask because I wonder if in the future we should refactor the `domain` logic such that we only calculate what is needed.

Reply:
good question. It looks like no right now but actually yes! I'm planning to have the component decide if it should show truncation indicators, so it will need to know if the set axis range is inside or outside the data range. I'll add a comment!

Reply:
ooh okay no you're right we don't need to calculate xmin or xmax if we have the range passed in. We already do the data range part elsewhere. Good catch!